packages/common/src/components/ActionServiceDropdown/ActionServiceDropdown.tsx

```diff
@@ -73,6 +73,18 @@ const ActionsAsDropdown = ({
     ) : (
       <DropdownToggle onToggle={setIsActionMenuOpen}>{dropdownToggleText}</DropdownToggle>
     );
+
+  const onClick = (
```

Comment:
```suggestion
  const onClick: (
  event: MouseEvent | React.MouseEvent<unknown, MouseEvent> | React.KeyboardEvent<Element>,
) => void | undefined = typeof cta === 'function' ? cta : () => cta.href && history.push(cta.href);
```

Reply:
The `cta` parameter is a internal `Array.map` iterator so a solution for that is a bit different.

Anyway for this specific case we can skip specifying the function type (e.g. 
```

  event: MouseEvent | React.MouseEvent<unknown, MouseEvent> | React.KeyboardEvent<Element>,
) => void )
```
since there is no function parameter used for this specific handler call and no return value,  so in case of function signature changes in PF5 then no errors will be triggered here.